[2024-07-14T04:16:53.806+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-14T04:16:53.980+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ELT-PROJECT.transform.dim_employees.test scheduled__2024-06-29T00:00:00+00:00 [queued]>
[2024-07-14T04:16:54.039+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ELT-PROJECT.transform.dim_employees.test scheduled__2024-06-29T00:00:00+00:00 [queued]>
[2024-07-14T04:16:54.040+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-14T04:16:54.105+0000] {taskinstance.py:2330} INFO - Executing <Task(DbtTestLocalOperator): transform.dim_employees.test> on 2024-06-29 00:00:00+00:00
[2024-07-14T04:16:54.144+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'ELT-PROJECT', 'transform.dim_employees.test', 'scheduled__2024-06-29T00:00:00+00:00', '--job-id', '1226', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmp9b491uio']
[2024-07-14T04:16:54.153+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=15396) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-14T04:16:54.178+0000] {standard_task_runner.py:91} INFO - Job 1226: Subtask transform.dim_employees.test
[2024-07-14T04:16:54.178+0000] {standard_task_runner.py:63} INFO - Started process 15412 to run task
[2024-07-14T04:16:54.329+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-07-14T04:16:54.584+0000] {task_command.py:426} INFO - Running <TaskInstance: ELT-PROJECT.transform.dim_employees.test scheduled__2024-06-29T00:00:00+00:00 [running]> on host airflow-scheduler
[2024-07-14T04:16:55.389+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Data-Ninja' AIRFLOW_CTX_DAG_ID='ELT-PROJECT' AIRFLOW_CTX_TASK_ID='transform.dim_employees.test' AIRFLOW_CTX_EXECUTION_DATE='2024-06-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-29T00:00:00+00:00'
[2024-07-14T04:16:55.392+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-14T04:16:55.430+0000] {local.py:174} INFO - Could not import dbtRunner. Falling back to subprocess for invoking dbt.
[2024-07-14T04:16:55.434+0000] {local.py:292} INFO - Cloning project to writable temp directory /tmp/tmp4_k_57jy from /opt/airflow/dags/dbt/dbt_project
[2024-07-14T04:16:55.456+0000] {local.py:303} INFO - Partial parse is enabled and the latest partial parse file is /tmp/cosmos/ELT-PROJECT__transform/target/partial_parse.msgpack
[2024-07-14T04:16:55.606+0000] {base.py:84} INFO - Using connection ID 'snow_conn' for task execution.
[2024-07-14T04:16:55.609+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2024-07-14T04:16:55.627+0000] {config.py:298} INFO - Profile caching is enable.
[2024-07-14T04:16:55.638+0000] {config.py:272} INFO - Profile found in cache using profile: /tmp/cosmos/profile/a0017dab0633177292578162811b91dcb9c56d6e54017e9660056cc364a762ad/profiles.yml.
[2024-07-14T04:16:55.640+0000] {local.py:247} INFO - Trying to run the command:
 ['/opt/airflow/dbt_venv/bin/dbt', 'deps', '--project-dir', '/tmp/tmp4_k_57jy', '--profiles-dir', '/tmp/cosmos/profile/a0017dab0633177292578162811b91dcb9c56d6e54017e9660056cc364a762ad', '--profile', 'dbt_project', '--target', 'dev']
From /tmp/tmp4_k_57jy
[2024-07-14T04:16:55.643+0000] {subprocess.py:60} INFO - Tmp dir root location: 
 /tmp
[2024-07-14T04:16:55.645+0000] {subprocess.py:73} INFO - Running command: ['/opt/airflow/dbt_venv/bin/dbt', 'deps', '--project-dir', '/tmp/tmp4_k_57jy', '--profiles-dir', '/tmp/cosmos/profile/a0017dab0633177292578162811b91dcb9c56d6e54017e9660056cc364a762ad', '--profile', 'dbt_project', '--target', 'dev']
[2024-07-14T04:16:55.748+0000] {subprocess.py:84} INFO - Command output:
[2024-07-14T04:17:11.555+0000] {subprocess.py:94} INFO - [0m04:17:11  Running with dbt=1.8.3
[2024-07-14T04:17:13.013+0000] {subprocess.py:94} INFO - [0m04:17:13  Updating lock file in file path: /tmp/tmp4_k_57jy/package-lock.yml
[2024-07-14T04:17:13.067+0000] {subprocess.py:94} INFO - [0m04:17:13  Installing dbt-labs/dbt_utils
[2024-07-14T04:17:14.037+0000] {subprocess.py:94} INFO - [0m04:17:14  Installed from version 0.8.6
[2024-07-14T04:17:14.045+0000] {subprocess.py:94} INFO - [0m04:17:14  Updated version available: 1.2.0
[2024-07-14T04:17:14.053+0000] {subprocess.py:94} INFO - [0m04:17:14
[2024-07-14T04:17:14.061+0000] {subprocess.py:94} INFO - [0m04:17:14  Updates available for packages: ['dbt-labs/dbt_utils']
[2024-07-14T04:17:14.067+0000] {subprocess.py:94} INFO - Update your versions in packages.yml, then run dbt deps
[2024-07-14T04:17:16.135+0000] {subprocess.py:98} INFO - Command exited with return code 0
[2024-07-14T04:17:16.140+0000] {local.py:254} INFO - Update your versions in packages.yml, then run dbt deps
[2024-07-14T04:17:16.143+0000] {local.py:247} INFO - Trying to run the command:
 ['/opt/airflow/dbt_venv/bin/dbt', 'test', '--models', 'dim_employees', '--project-dir', '/tmp/tmp4_k_57jy', '--profiles-dir', '/tmp/cosmos/profile/a0017dab0633177292578162811b91dcb9c56d6e54017e9660056cc364a762ad', '--profile', 'dbt_project', '--target', 'dev']
From /tmp/tmp4_k_57jy
[2024-07-14T04:17:16.157+0000] {subprocess.py:60} INFO - Tmp dir root location: 
 /tmp
[2024-07-14T04:17:16.160+0000] {subprocess.py:73} INFO - Running command: ['/opt/airflow/dbt_venv/bin/dbt', 'test', '--models', 'dim_employees', '--project-dir', '/tmp/tmp4_k_57jy', '--profiles-dir', '/tmp/cosmos/profile/a0017dab0633177292578162811b91dcb9c56d6e54017e9660056cc364a762ad', '--profile', 'dbt_project', '--target', 'dev']
[2024-07-14T04:17:16.235+0000] {subprocess.py:84} INFO - Command output:
[2024-07-14T04:17:31.224+0000] {subprocess.py:94} INFO - [0m04:17:31  Running with dbt=1.8.3
[2024-07-14T04:17:36.172+0000] {subprocess.py:94} INFO - [0m04:17:36  Registered adapter: snowflake=1.8.3
[2024-07-14T04:17:41.824+0000] {subprocess.py:94} INFO - [0m04:17:41  Found 20 models, 27 data tests, 11 sources, 684 macros
[2024-07-14T04:17:41.857+0000] {subprocess.py:94} INFO - [0m04:17:41
[2024-07-14T04:17:53.569+0000] {subprocess.py:94} INFO - [0m04:17:53  Concurrency: 1 threads (target='dev')
[2024-07-14T04:17:53.574+0000] {subprocess.py:94} INFO - [0m04:17:53
[2024-07-14T04:17:53.695+0000] {subprocess.py:94} INFO - [0m04:17:53  1 of 2 START test not_null_dim_employees_EMPLOYEEID ............................ [RUN]
[2024-07-14T04:17:55.938+0000] {subprocess.py:94} INFO - [0m04:17:55  1 of 2 PASS not_null_dim_employees_EMPLOYEEID .................................. [[32mPASS[0m in 2.25s]
[2024-07-14T04:17:55.952+0000] {subprocess.py:94} INFO - [0m04:17:55  2 of 2 START test unique_dim_employees_EMPLOYEEID .............................. [RUN]
[2024-07-14T04:17:57.829+0000] {subprocess.py:94} INFO - [0m04:17:57  2 of 2 PASS unique_dim_employees_EMPLOYEEID .................................... [[32mPASS[0m in 1.87s]
[2024-07-14T04:17:57.860+0000] {subprocess.py:94} INFO - [0m04:17:57
[2024-07-14T04:17:57.862+0000] {subprocess.py:94} INFO - [0m04:17:57  Finished running 2 data tests in 0 hours 0 minutes and 15.97 seconds (15.97s).
[2024-07-14T04:17:58.305+0000] {subprocess.py:94} INFO - [0m04:17:58
[2024-07-14T04:17:58.310+0000] {subprocess.py:94} INFO - [0m04:17:58  [32mCompleted successfully[0m
[2024-07-14T04:17:58.313+0000] {subprocess.py:94} INFO - [0m04:17:58
[2024-07-14T04:17:58.317+0000] {subprocess.py:94} INFO - [0m04:17:58  Done. PASS=2 WARN=0 ERROR=0 SKIP=0 TOTAL=2
[2024-07-14T04:18:01.038+0000] {subprocess.py:98} INFO - Command exited with return code 0
[2024-07-14T04:18:01.047+0000] {local.py:254} INFO - [0m04:17:58  Done. PASS=2 WARN=0 ERROR=0 SKIP=0 TOTAL=2
[2024-07-14T04:18:02.141+0000] {local.py:196} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/manifest/v12.json is above dbt-ol supported version 7. This might cause errors.
[2024-07-14T04:18:02.144+0000] {local.py:196} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/run-results/v6.json is above dbt-ol supported version 5. This might cause errors.
[2024-07-14T04:18:02.158+0000] {local.py:349} INFO - Inlets: [Dataset(uri='snowflake://OU00370.europe-west2.gcp/ELT_BATCH.dwh.dim_employees', extra=None)]
[2024-07-14T04:18:02.175+0000] {local.py:350} INFO - Outlets: []
[2024-07-14T04:18:02.177+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-07-14T04:18:02.521+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-14T04:18:02.659+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ELT-PROJECT, task_id=transform.dim_employees.test, run_id=scheduled__2024-06-29T00:00:00+00:00, execution_date=20240629T000000, start_date=20240714T041653, end_date=20240714T041802
[2024-07-14T04:18:02.819+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-14T04:18:03.029+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-14T04:18:03.041+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
